crates/forge-runner/src/lib.rs

```diff
@@ -38,6 +40,7 @@ mod printing;
 mod running;
 
 pub const CACHE_DIR: &str = ".snfoundry_cache";
+pub const SIERRA_TEST_CODE_DIR: &str = ".snfoundry_test_code";
```

Comment:
This copied sierras should are part of the trace, this name should be in `build_trace_data` module.

Reply:
Done